webgoat-integration-tests/src/test/java/org/owasp/webgoat/IntegrationTest.java

```diff
@@ -0,0 +1,200 @@
+package org.owasp.webgoat;
+
+import io.restassured.RestAssured;
+import io.restassured.config.RestAssuredConfig;
+import io.restassured.config.SSLConfig;
+import lombok.Getter;
+import org.hamcrest.CoreMatchers;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.owasp.webwolf.WebWolf;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.util.Map;
+import java.util.UUID;
+
+public abstract class IntegrationTest {
+
+    private static String WEBGOAT_URL = "http://localhost:8080/WebGoat/";
```

Comment:
Do you want to run the integration tests always on these fixed ports? Or is this temporary so we can later on use random ports for this if not specified by some -D command line parameter

Reply:
The last part was the idea, for now have it run on 8080 (with a check if it already runs as standalone) and then use that. In the future we can extend to run it on different ports, with the args it should be quite simple to accomplish this

Reply:
ok fine